### PR TITLE
Various technical fixes and style guide changes in Data Explorer

### PR DESCRIPTION
**Pull request by JFix89**
Hi John,

Thanks again for taking a look over this. I've made quite a few changes to the Data Explorer to improve the look of several charts with the new PHS colour palette. I've also made style guide changes to the UI text and dropdown menus and made technical tweaks (e.g. changing the tooltip for the CBF diagram and reordering quarters in the dropdowns so they appear chronologically). Please let me know if you have any questions on any of this - happy to have a call and demo things in the Data Explorer if that would help too.

Thanks,

James

### CODE REVIEW

File: Data-Explorer/ui.R [hunk 1]
```diff
@@ -97,46 +97,47 @@ ui_code <- tagList( #needed for shinyjs
...
-                              "for information on COVID-19 in Scotland "
+                                "Scottish Government website (external website)",target = "_blank",class = "special_link"),
+                              " for information on COVID-19 in Scotland"
                               ),
                       tags$li("there are issues with the quality of the data",
                               "presented, which are summarised in the",
                               tags$a(
                                 href = paste0(pub_url, "data-quality/"),
                                 "Data quality", target = "_blank",
-                                class = "special-link"), " and ",
```
Comment (JohnPercivalPHS) [inline Data-Explorer/ui.R:120]: Change back.

File: Data-Explorer/ui.R [hunk 2]
```diff
@@ -97,46 +97,47 @@ ui_code <- tagList( #needed for shinyjs
...
+                                class = "special_link"), "and",
                               tags$a(
                                 href = paste0(pub_url, "trend-data/"),
                                 "Trend data", target = "_blank",
-                                class = "special-link"), "sections"),
```
Comment (JohnPercivalPHS) [inline Data-Explorer/ui.R:124]: Change back.

File: Data-Explorer/ui.R [hunk 3]
```diff
@@ -327,14 +327,14 @@ ui_code <- tagList( #needed for shinyjs
...
 ### Tab 5: Deprivation (SIMD) ----
     tabPanel("Deprivation",
-             icon = icon("user-circle-o"),
+             icon = icon("circle-user", lib = "font-awesome"),
              h3("Deprivation"),
              p("In this section, you can explore the data by
         different levels of deprivation in accordance with the",
                tags$a(href="https://gov.scot/simd",
-                      "Scottish Index of Multiple Deprivation (SIMD) (external website)" , target = "_blank",
-                      class = "special-link"),
```
Comment (JohnPercivalPHS) [inline Data-Explorer/ui.R:336]: Change back.

File: Data-Explorer/ui.R [hunk 4]
```diff
@@ -475,23 +478,22 @@ ui_code <- tagList( #needed for shinyjs
...
                ‘Download data’ button."),
              uiOutput("table_notes"),
              p("For more information on the data, data variables and definitions,
            please see the ",
                tags$a(
                  href = paste0(pub_url, "data-files/"),
-                 "Data Files", target = "_blank",
-                 class = "special-link"),
+                 "Data files", target = "_blank",
+                 class = "special_link"),
```
Comment (JohnPercivalPHS) [inline Data-Explorer/ui.R:491]: change special_link back to special-link

File: Data-Explorer/ui.R [hunk 5]
```diff
@@ -475,23 +478,22 @@ ui_code <- tagList( #needed for shinyjs
...
                "and",
                tags$a(
                  href = paste0(pub_url, "glossary/"),
                  "Glossary", target = "_blank",
-                 class = "special-link"),
```
Comment (JohnPercivalPHS) [inline Data-Explorer/ui.R:494]: Change back.

**Review by JohnPercivalPHS — CHANGES_REQUESTED** [review]
Hi James,

The text changes and updated colour palette look great.

However, we need the html tags class names to match those in the css.

Thanks,

John

File: Data-Explorer/global.R [hunk 6]
```diff
@@ -63,24 +63,57 @@ pub_url = paste0("https://publichealthscotland.scot/publications/",
 data_bed <- readRDS(paste0(rds_filepath, "beds.rds"))
 
 # Specialty data
-data_spec <- readRDS(paste0(rds_filepath, "spec.rds"))
+data_spec <- readRDS(paste0(rds_filepath, "spec.rds")) %>% 
+    mutate(file = case_when(file %in% "Inpatients/Day Cases" ~ "Inpatients/day cases",
```
Comment (JohnPercivalPHS) [inline Data-Explorer/global.R:67]: These should be changed in functions.R like the rest of the naming changes.

**Review by JohnPercivalPHS — CHANGES_REQUESTED** [review]
Hi James,

Please can you move the naming changes to functions.R?

Thanks,

John

**Comment by JFix89** [comment]
Hi John,

I've sent a separate email on this a moment ago - agree that it would be better to do as you suggest to make scripts consistent, but can this be put off until after the publication and added to the development tab? I'll gladly pick it up then (along with some other outstanding improvements) but conscious of deadlines for the data explorer at the moment and not keen on tinkering with the code just now when everything is working as it should.

Thanks,

James

**Review by JohnPercivalPHS — APPROVED** [review]
Hi, 

All looks good and thanks for adding the new colour pallette scheme.

John

**Pull request merged**